partner_supplier_invoice_smartbutton/__openerp__.py

```diff
@@ -35,7 +35,7 @@
         'views/partner_view.xml',
     ],
     'depends': [
-        'account',
+        'purchase',
     ],
```

Comment:
@PeterAlabaster it looks like this module leans directly on accounting code, so should probably list 'account' as an explicit depends too.

Reply:
Thanks Nick, didn't think of that!